README.md

```diff
@@ -4,11 +4,28 @@
 
 The official repository for the iGenVar project.
 
+iGenVar is intended to be a caller for all types of genetic variation: SNPs, indels and larger structural variations
+(insertions, deletions, inversions, translocations, CNVs, nested SVs).
+It uses both Illumina short reads and PacBio long reads for this purpose.
+
+David Heller in the Vingron lab of the MPI-MG and Tim White in the Kehr lab at BIH have both developed an SV caller for
+long read sequencing data. Instead of competing with each other, we want to join forces and combine the two tools, SVIM
+and SVIRL, into one better and more versatile tool.
+On the other hand, there were some tool developments in the Reinert Lab (FU Berlin): Vaquita, a short read SV caller,
+SViper, a refinement tool and Vaquita-LR a further development of Vaquita for long reads.
+We want to combine these approaches and use the SeqAn3 library as a basis for this new tool.
+
+## Current status:
+
+We can call insertions and deletions from long read data (SVIM methods implemented).
```

Comment:
Just as an idea, you could link your picture of your plan somewhere? (The one you showed last week), so a user knows what will go in next. :)